lottery/storage/mod.go

```diff
@@ -0,0 +1,7 @@
+package storage
+
+// KeyValue defines a simple key value storage
+type KeyValue interface {
+	Store(key, value []byte) error
```

Comment:
You can use _serde.Message_ instead so that in-mem implementations don't need to serialize.